newIDE/app/src/Utils/GDevelopServices/Extension.js

```diff
@@ -80,7 +111,9 @@ type SerializedExtensionWithTagsAsString = {
  * Transform the tags from their old representation sent by the API (a string)
  * to their new representation (array of strings).
  */
-const transformTagsAsStringToTagsAsArray = <T: { tags: string }>(
+const transformTagsAsStringToTagsAsArray = <
+  T: { tags: string } | { tags: string[] }
```

Comment:
If I understand correctly, you're making this function accept the new and old representation?

Reply:
Indeed